### ISSUE TITLE
Documentation of component sizes and how it should be used 

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystemet#1763]
Should be explained here: https://www.designsystemet.no/grunnleggende/designelementer/storrelser-og-avstander

https://pr-3083.designsystemet.no/grunnleggende/designelementer/storrelser-og-avstander

**Comment by mimarz** [comment]
Is this still relevant @Thunear @mrosvik ?

**Comment by mrosvik** [comment]
Yes, it is still relevant to explain when small, medium and large should be used, and how to combine them in different contexts.
